src/test/java/com/opentable/kafka/util/OffsetMetricsTest.java

```diff
@@ -0,0 +1,82 @@
+package com.opentable.kafka.util;
+
+import java.time.Duration;
+import java.util.Collections;
+
+import com.codahale.metrics.Counting;
+
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.junit.Test;
+
+import com.opentable.kafka.embedded.EmbeddedKafkaBroker;
+
+public class OffsetMetricsTest {
+    private static final String GROUP_ID = "group-1";
+
+    @Test(expected = IllegalArgumentException.class)
+    public void testNoTopics() throws InterruptedException {
+        try (EmbeddedKafkaBroker ekb = TestUtils.broker()) {
```

Comment:
Could factor the broker out into a `@Rule` to reduce the amount of code per test case